edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.E2E.Test/AgentTests.cs

```diff
@@ -181,7 +181,7 @@ public async Task AgentStartsUpModules(TestConfig testConfig)
                     validated = testConfig.Validator.Validate();
                     if (!validated)
                     {
-                        Thread.Sleep(TimeSpan.FromSeconds(5));
+                        Thread.Sleep(TimeSpan.FromSeconds(10));
```

Comment:
>romSeconds(10)); [](start = 47, length = 16)

This is not explained on the description. Can you add a line there explaining why changing from 5 to 10?

Reply:
reverted since this test project will be run individually.